lib/mintel/dashboards/services/omni/analytics.libsonnet

```diff
@@ -0,0 +1,162 @@
+local layout = import 'components/layout.libsonnet';
+local commonPanels = import 'components/panels/common.libsonnet';
+local promQuery = import 'components/prom_query.libsonnet';
+{
+
+  apiServiceLatency(serviceSelectorKey='service', serviceSelectorValue='$service', span=12)::
+    local config = {
+      serviceSelectorKey: serviceSelectorKey,
+      serviceSelectorValue: serviceSelectorValue,
+    };
+
+    layout.grid([
+
+    commonPanels.latencyTimeseries(
+      title='API Server Latency',
+      description='Percentile Latency For Analytics API Server',
+      yAxisLabel='Time',
+      format='s',
+      legend_show=true,
+      span=span,
+      height=300,
+      query=|||
+        histogram_quantile(0.95,
+          sum(
+            rate(
+              http_request_duration_seconds_bucket{namespace="$namespace", analytics_type=~"$analytics_type", %(serviceSelectorKey)s="%(serviceSelectorValue)s"}[$__interval]))
```

Comment:
we should pre-record this , like we did for haproxy or it will fail when the time range is too big

Reply:
I was intending todo all the pre-recordings in a different MR.

I think the impact is wider than just omni, since we should  also take into account the un-initialized metric.

Reply:
yeah the impact is weider , which is why is good to have common names for metrics where possible

Reply:
why do you want to do it in a separate MR and then have to change this dashboard ?

Reply:
Yeah ok I'll do the pre-recordings here